### ISSUE TITLE
Why we should divide logits by temperature?

### ISSUE THREAD

**Issue by FiresWorker** [PhamNguyen97/TSA_pytorch#1]
Think you for your code so much.
I have a question about the paramter "temperature", why we should divide logits by temperature.

 logits = logits / self.temperature

In the paper, the author did't use this parameter.
Thank you very much for answering my question！
